### PR TITLE
Daily markdownlint

### PR DESCRIPTION
**Pull request by OnkarRuikar**

### CODE REVIEW

File: files/en-us/web/svg/tutorial/getting_started/index.md [hunk 1]
```diff
@@ -48,8 +48,8 @@ The rendering process involves the following:
 - SVG files on the web can be displayed directly in the browser or embedded in HTML files via several methods:
 
   - If the HTML is XHTML and is delivered as type `application/xhtml+xml`, the SVG can be directly embedded in the XML source.
-  - The SVG can also be directly embedded in HTML. 
-  - An `img` element can be used. . 
+  - The SVG can also be directly embedded in HTML.
+  - An `img` element can be used. .
```
Comment (Josh-Cena) [inline files/en-us/web/svg/tutorial/getting_started/index.md:52]:
```suggestion
  - An `img` element can be used.
```

File: files/en-us/web/svg/tutorial/svg_in_html_introduction/index.md [hunk 2]
```diff
@@ -38,25 +38,25 @@ To include an inline SVG in an HTML file, paste the entire SVG file into the HTM
 
 ## Discussion
 
-The page is regular HTML and CSS with a single SVG. The only interesting part is the `<svg>` element it contains. This element and its children are declared to be in the SVG namespace. The element contains a gradient and two shapes filled with the gradient. The gradient color stops have their colors set by CSS. 
+The page is regular HTML and CSS with a single SVG. The only interesting part is the `<svg>` element it contains. This element and its children are declared to be in the SVG namespace. The element contains a gradient and two shapes filled with the gradient. The gradient color stops have their colors set by CSS.
 
 There are three attributes and one nested element worth noting:
 
-1.  The [`viewBox`](/en-US/docs/Web/SVG/Attribute/viewBox) attribute establishes a logical coordinate system which the SVG picture's coordinates are relative to. In this case our picture is laid out in a 100 by 100 viewport.
+1. The [`viewBox`](/en-US/docs/Web/SVG/Attribute/viewBox) attribute establishes a logical coordinate system which the SVG picture's coordinates are relative to. In this case our picture is laid out in a 100 by 100 viewport.
```
Comment (Josh-Cena) [inline files/en-us/web/svg/tutorial/svg_in_html_introduction/index.md:46]: Will removing the line break here make a difference? (I know in some renderers it does)

Comment (OnkarRuikar) [reply files/en-us/web/svg/tutorial/svg_in_html_introduction/index.md:46]: Extra line in between the list items doesn't break the list. You can check the rendered output in preview URLs provided by the PR companion.

**Review by teoli2003 — APPROVED** [review]
👍

**Pull request merged**